Lombiq.Tests.UI/Services/OrchardCoreInstance.cs

```diff
@@ -212,6 +212,7 @@ private async Task StopOrchardAppAsync()
 
         await _orchardApplication.DisposeAsync();
         _orchardApplication = null;
+        await OrchardCoreInstanceCounter.PortLeases.StopLeaseAsync(_url.Port, CancellationToken.None);
```

Comment:
Wait, so the OC ports were never actually released?

Reply:
Yeah :(